src/student/student.service.ts

```diff
@@ -19,15 +20,30 @@ export class StudentService {
     return `This action returns all student`;
   }
 
-  findOne(id: number) {
-    return `This action returns a #${id} student`;
-  }
-
   update(id: number, updateStudentDto: UpdateStudentDto) {
     return `This action updates a #${id} student`;
   }
 
   remove(id: number) {
     return `This action removes a #${id} student`;
   }
+
+  async getStudentByUuid(uuid: string): Promise<ReadStudentDto> {
```

Comment:
hi ya, pwede pa apil ug return sa data for event and sa payment na connected sa student. thank you!

Reply:
return lang nako ang eventid ug paymentid or do u want na ang unod sa event og payment pod ang ireturn?

Reply:
apila nalang ang content mismo ya